contrib/devtools/gen-manpages.py

```diff
@@ -42,7 +42,7 @@
         print(f'{abspath} not found or not an executable', file=sys.stderr)
         sys.exit(1)
     # take first line (which must contain version)
-    verstr = r.stdout.split('\n')[0]
+    verstr = r.stdout.splitlines()[0]
     # last word of line is the actual version e.g. v22.99.0-5c6b3d5b3508
     verstr = verstr.split()[-1]
     assert verstr.startswith('v')
```

Comment:
The lines below here can go now ("Only bitcoin-qt prints the copyright message on --version, so store it specifically."). Could even store the copyright message per binary like the other version info.

Reply:
Thanks, should be addressed.